docs/csharp/tutorials/inheritance.md

```diff
@@ -0,0 +1,294 @@
+
+- Publisher-related members
+
+  Two read-only properties, `Publisher` and `Type`, return the value of the private `pubName` and `pubType` fields. The values are originally supplied by the call to the `Publication` class constructor.
+
+- Publishing-related members
+
+  Two methods, `Publish` and `GetPublicationDate`, set and return the publication date. The `Publish` method sets a private  `published` flag to `true` when it is called and assigns the date passed to it as an argument to the private `datePublished` field. The `GetPublicationDate` method returns the string "NYP" if the `published` flag is `false`, and the value of the `datePublished` field if it is `true`.
+
+- Copyright-related members
+
+  The `Copyright` method takes the name of the copyright holder and the year of the copyright as arguments and assigns them to the private `copyrName` and `copyrDate` fields. The values can be retrieved from the `CopyrightName` and `CopyrightDate` properties.
+
+- An override of the `ToString` method
+
+  If @System.Object.ToString is inherited, it returns the fully qualified name of the type, which is of little use in differentiating one instance from another. The `Publication` class overrides @System.Object.ToString to return the value of the `Title` property.
+
+The following figure illustrates the relationship between our base `Publication` class and its implicitly inherited @System.Object class.
+
+![The Object and Publication classes](media/publication-class.jpg)
+
+### The `Book` class ###
+
+The `Book` class represents a book as a specialized type of publication. The following example shows the source code for the `Book` class.
+
+[!CODE [Inheritance](../../../samples/snippets/csharp/tutorials/inheritance/base-and-derived.cs#2)]
+
+In addition to the members that it inherits from `Publication`, the `Book` class defines the following unique members and member overrides:
+
+- Two constructors
+
+  The two `Book` constructors share three common parameters. Two, *title* and *publisher*, correspond to parameters of the `Publication` constructor. The third is *author*, which is stored to a private `authorName` field. One constructor includes an *isbn* parameter, which is stored to the private `id` field.
+
+  The first constructor uses the [this](../language-reference/keywords/this.md) keyword to call the other constructor. This is a common pattern in defining constructors; constructors with fewer parameters provide default values when calling the constructor with the greatest number of parameters.
+
+  The second constructor uses the [base](../language-reference/keywords/base.md) keyword to pass the title and publisher name to the base class constructor. A derived class constructor should always call a constructor of its base class. If you don't make an explicit call to a base class constructor in your source code, the C# compiler automatically supplies a call to the base class' default or parameterless constructor.
```

Comment:
I would delete the sentence "A derived class constructor should always call a constructor of its base class". As you point out, it always does anyway. You just need to call a constructor if you want a different one called.